internal/anelace/argparse.go

```diff
@@ -120,6 +120,13 @@ func NewFromArgv(argv []string) (anl *Anelace) {
 				emRootsJsonl:  nil,
 				emCarV1Stream: nil,
 			},
+
+			// some opinionated defaults
+			requestedNodeEncoder: "unixfsv1_merkledag-compat-protobuf",
+			requestedChunker:     "fixed-size_1048576",                                     // 1 MiB static chunking
+			requestedCollector:   "trickle_max-direct-leaves=4096_max-sibling-subgroups=8", // trickledag with 4096 MaxDirectLeaves + 8 MaxSiblingSubgroups
```

Comment:
Now that I think about it 4096 * ~40 is a bit much ( 160k )
Let's make it 2048 instead

Reply:
Is there a way to actually measure this and then decide?